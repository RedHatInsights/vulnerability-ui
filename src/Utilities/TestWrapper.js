import propTypes from 'prop-types';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations';

const TestWrapper = ({ children, store }) => {
    return (
        <IntlProvider locale="en">
            { store ?
                <Provider store={store}>
                    <Router>
                        {children}
                    </Router>
                </Provider> : children
            }
        </IntlProvider>
    );
};

TestWrapper.propTypes = {
    children: propTypes.node,
    store: propTypes.object
};

export default TestWrapper;
