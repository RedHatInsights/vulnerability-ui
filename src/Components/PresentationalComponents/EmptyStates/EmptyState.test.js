import {
    EmptyStateNoCVEs,
    EmptyStateNoSystems,
    EmptyStateExcludedSystem
} from './EmptyStates'
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import TestWrapper from '../../../Utilities/TestWrapper';
import messages from '../../../Messages'

describe('Empty states', () => {
    it('should render EmptyStateNoCVEs', () => {
        const { asFragment } = render(<TestWrapper><EmptyStateNoCVEs secondParagraph={messages.emptyStateThereShouldBeCVEs}/></TestWrapper>);
        expect(asFragment()).toMatchSnapshot();

        expect(screen.getByTestId('prod-sec-link')).toBeTruthy();
    })

    it('should render EmptyStateNoSystems', () => {
        const { asFragment } = render(<TestWrapper><EmptyStateNoSystems /></TestWrapper>);
        expect(asFragment()).toMatchSnapshot();
    })

    it('should render EmptyStateExcludedSystem', () => {
        const mockAction = jest.fn();
        const { asFragment } = render(<TestWrapper><EmptyStateExcludedSystem buttonAction={mockAction} /></TestWrapper>);
        expect(asFragment()).toMatchSnapshot();

        userEvent.click(screen.getByTestId('empty-state-button-action'));
        expect(mockAction).toHaveBeenCalled();
    })
})
