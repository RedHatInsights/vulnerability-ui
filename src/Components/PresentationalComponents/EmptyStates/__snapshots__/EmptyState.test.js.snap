// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty state should render EmptyCVEList 1`] = `
<IntlProvider
  locale="en"
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="en"
    messages={
      Object {
        "default.cancel": "Cancel",
        "default.delete": "Delete",
        "default.remove": "Remove",
        "default.save": "Save",
      }
    }
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <Component
      style={
        Object {
          "height": "auto",
        }
      }
    >
      <div
        className="pf-l-bullseye"
        style={
          Object {
            "height": "auto",
          }
        }
      >
        <Component
          variant="large"
        >
          <div
            className="pf-c-empty-state pf-m-lg"
          >
            <div
              className="pf-c-empty-state__content"
            >
              <Component
                headingLevel="h5"
                size="lg"
              >
                <h5
                  className="pf-c-title pf-m-lg"
                >
                  <FormattedMessage
                    defaultMessage="No CVEs reported for connected systems"
                    id="emptyState.noReportedSystem"
                    values={Object {}}
                  >
                    No CVEs reported for connected systems
                  </FormattedMessage>
                </h5>
              </Component>
              <Component>
                <div
                  className="pf-c-content"
                >
                  <Component
                    style={
                      Object {
                        "whiteSpace": "normal",
                      }
                    }
                  >
                    <div
                      className="pf-c-empty-state__body"
                      style={
                        Object {
                          "whiteSpace": "normal",
                        }
                      }
                    >
                      <Component>
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="No published CVEs in the Vulnerability service affect your connected systems"
                            id="emptyState.noCveAffects"
                            values={Object {}}
                          >
                            No published CVEs in the Vulnerability service affect your connected systems
                          </FormattedMessage>
                        </p>
                      </Component>
                      <FormattedMessage
                        defaultMessage="If you think your systems have applicable CVEs, or would like more information, please contact   "
                        id="emptyState.ifCveForSystems"
                        values={Object {}}
                      >
                        If you think your systems have applicable CVEs, or would like more information, please contact   
                      </FormattedMessage>
                      <a
                        href="https://access.redhat.com/security/team/contact/"
                        rel="noopener noreferrer"
                        target="__blank"
                      >
                        <FormattedMessage
                          defaultMessage="Red Hat Products Security"
                          id="emptyState.forMoreLink"
                          values={Object {}}
                        >
                          Red Hat Products Security
                        </FormattedMessage>
                         
                        <ExternalLinkAltIcon
                          color="currentColor"
                          noVerticalAlign={false}
                          size="sm"
                        >
                          <svg
                            aria-hidden={true}
                            aria-labelledby={null}
                            fill="currentColor"
                            height="1em"
                            role="img"
                            style={
                              Object {
                                "verticalAlign": "-0.125em",
                              }
                            }
                            viewBox="0 0 512 512"
                            width="1em"
                          >
                            <path
                              d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                              transform=""
                            />
                          </svg>
                        </ExternalLinkAltIcon>
                      </a>
                      .
                    </div>
                  </Component>
                </div>
              </Component>
            </div>
          </div>
        </Component>
      </div>
    </Component>
  </IntlProvider>
</IntlProvider>
`;

exports[`Empty state should render EmptyCVEListForSystem 1`] = `
<IntlProvider
  locale="en"
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="en"
    messages={
      Object {
        "default.cancel": "Cancel",
        "default.delete": "Delete",
        "default.remove": "Remove",
        "default.save": "Save",
      }
    }
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <Component
      style={
        Object {
          "height": "auto",
        }
      }
    >
      <div
        className="pf-l-bullseye"
        style={
          Object {
            "height": "auto",
          }
        }
      >
        <Component
          variant="large"
        >
          <div
            className="pf-c-empty-state pf-m-lg"
          >
            <div
              className="pf-c-empty-state__content"
            >
              <Component
                headingLevel="h5"
                size="lg"
              >
                <h5
                  className="pf-c-title pf-m-lg"
                >
                  <FormattedMessage
                    defaultMessage="No CVEs reported for connected systems"
                    id="emptyState.noReportedSystem"
                    values={Object {}}
                  >
                    No CVEs reported for connected systems
                  </FormattedMessage>
                </h5>
              </Component>
              <Component>
                <div
                  className="pf-c-content"
                >
                  <Component
                    style={
                      Object {
                        "whiteSpace": "normal",
                      }
                    }
                  >
                    <div
                      className="pf-c-empty-state__body"
                      style={
                        Object {
                          "whiteSpace": "normal",
                        }
                      }
                    >
                      <Component>
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="No published CVEs in the Vulnerability service affect your connected systems"
                            id="emptyState.noCveAffects"
                            values={Object {}}
                          >
                            No published CVEs in the Vulnerability service affect your connected systems
                          </FormattedMessage>
                        </p>
                      </Component>
                      <Component
                        component="p"
                      >
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="If you think this system has applicable CVEs, or would like more information, please contact   "
                            id="emptyState.ifCveForSystem"
                            values={Object {}}
                          >
                            If you think this system has applicable CVEs, or would like more information, please contact   
                          </FormattedMessage>
                          <a
                            href="https://access.redhat.com/security/team/contact/"
                            rel="noopener noreferrer"
                            target="__blank"
                          >
                            <FormattedMessage
                              defaultMessage="Red Hat Products Security"
                              id="emptyState.forMoreLink"
                              values={Object {}}
                            >
                              Red Hat Products Security
                            </FormattedMessage>
                              
                            <ExternalLinkAltIcon
                              color="currentColor"
                              noVerticalAlign={false}
                              size="sm"
                            >
                              <svg
                                aria-hidden={true}
                                aria-labelledby={null}
                                fill="currentColor"
                                height="1em"
                                role="img"
                                style={
                                  Object {
                                    "verticalAlign": "-0.125em",
                                  }
                                }
                                viewBox="0 0 512 512"
                                width="1em"
                              >
                                <path
                                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                                  transform=""
                                />
                              </svg>
                            </ExternalLinkAltIcon>
                          </a>
                          .
                        </p>
                      </Component>
                    </div>
                  </Component>
                </div>
              </Component>
            </div>
          </div>
        </Component>
      </div>
    </Component>
  </IntlProvider>
</IntlProvider>
`;

exports[`Empty state should render EmptyExcludedSystem 1`] = `
<IntlProvider
  locale="en"
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="en"
    messages={
      Object {
        "default.cancel": "Cancel",
        "default.delete": "Delete",
        "default.remove": "Remove",
        "default.save": "Save",
      }
    }
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <EmptyExcludedSystem
      buttonAction={[MockFunction]}
    >
      <Component>
        <div
          className="pf-l-bullseye"
        >
          <Component
            variant="large"
          >
            <div
              className="pf-c-empty-state pf-m-lg"
            >
              <div
                className="pf-c-empty-state__content"
              >
                <Component
                  icon={[Function]}
                  size="sm"
                >
                  <SecurityIcon
                    aria-hidden="true"
                    className="pf-c-empty-state__icon"
                    color="currentColor"
                    noVerticalAlign={false}
                    size="sm"
                  >
                    <svg
                      aria-hidden="true"
                      aria-labelledby={null}
                      className="pf-c-empty-state__icon"
                      fill="currentColor"
                      height="1em"
                      role="img"
                      style={
                        Object {
                          "verticalAlign": "-0.125em",
                        }
                      }
                      viewBox="0 64 1024 1024"
                      width="1em"
                    >
                      <path
                        d="M925.714 938.194c-7.84 8.183-18.697 12.777-30.034 12.663h-767.68c-11.337 0.114-22.194-4.48-30.034-12.663-8.183-7.817-12.754-18.674-12.663-29.989v-511.817c0.137-38.949 7.726-77.509 22.331-113.623 13.851-35.657 32.457-69.257 55.314-99.909 23.497-30.72 49.806-59.177 78.629-84.96 26.72-24.503 54.857-47.406 84.297-68.571 25.76-18.286 52.64-35.497 80.64-51.657s47.863-27.154 59.634-32.983c11.771-5.783 21.211-10.217 28.32-13.326 10.926-5.326 23.726-5.326 34.651 0 7.109 3.040 16.549 7.497 28.32 13.326s31.68 16.777 59.703 32.846c28 16.206 54.88 33.44 80.64 51.657 29.44 21.166 57.577 44.069 84.297 68.571 28.823 25.829 55.131 54.331 78.606 85.097 22.857 30.674 41.463 64.297 55.314 99.954 14.606 36.114 22.194 74.674 22.331 113.623v511.771c0.114 11.314-4.457 22.171-12.617 29.989zM595.337 183.52c-0.206-15.291-12.549-27.634-27.84-27.84h-111.36c-15.291 0.206-27.634 12.549-27.84 27.84v97.44c0.206 15.291 12.549 27.634 27.84 27.84h111.36c15.291-0.206 27.634-12.549 27.84-27.84v-97.44zM596.206 429.097c-0.251-7.451-3.451-14.491-8.914-19.566-5.211-5.326-12.343-8.32-19.794-8.274h-111.36c-7.451-0.046-14.583 2.949-19.794 8.274-5.463 5.074-8.663 12.114-8.914 19.566l-12.183 334.057c-0.366 7.314 2.4 14.423 7.611 19.566 5.006 5.349 12.023 8.366 19.36 8.274h139.2c7.337 0.091 14.354-2.926 19.36-8.274 5.211-5.143 7.977-12.251 7.611-19.566l-12.183-334.057z"
                        transform="rotate(180 0 512) scale(-1 1)"
                      />
                    </svg>
                  </SecurityIcon>
                </Component>
                <Component
                  headingLevel="h5"
                  size="lg"
                >
                  <h5
                    className="pf-c-title pf-m-lg"
                  >
                    <FormattedMessage
                      defaultMessage="Excluded from vulnerability analysis"
                      id="systemExcludedFromAnalysis.title"
                      values={Object {}}
                    >
                      Excluded from vulnerability analysis
                    </FormattedMessage>
                  </h5>
                </Component>
                <Component>
                  <div
                    className="pf-c-empty-state__body"
                  >
                    <FormattedMessage
                      defaultMessage="This system has been excluded from vulnerability analysis.        It is not being evaluated for potential vulnerabilities by Red Hat Insights."
                      id="systemExcludedFromAnalysis.body"
                      values={Object {}}
                    >
                      This system has been excluded from vulnerability analysis.        It is not being evaluated for potential vulnerabilities by Red Hat Insights.
                    </FormattedMessage>
                  </div>
                </Component>
                <Component
                  component="a"
                  onClick={[MockFunction]}
                  variant="primary"
                >
                  <a
                    aria-disabled={false}
                    aria-label={null}
                    className="pf-c-button pf-m-primary"
                    data-ouia-component-id={null}
                    data-ouia-component-type="PF4/Button"
                    data-ouia-safe={true}
                    disabled={null}
                    onClick={[MockFunction]}
                    tabIndex={null}
                    type={null}
                  >
                    <FormattedMessage
                      defaultMessage="Resume vulnerability analysis"
                      id="systemExcludedFromAnalysis.resumeAnalysis"
                      values={Object {}}
                    >
                      Resume vulnerability analysis
                    </FormattedMessage>
                  </a>
                </Component>
              </div>
            </div>
          </Component>
        </div>
      </Component>
    </EmptyExcludedSystem>
  </IntlProvider>
</IntlProvider>
`;

exports[`Empty state should render EmptyVulnerabilityData 1`] = `
<IntlProvider
  locale="en"
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="en"
    messages={
      Object {
        "default.cancel": "Cancel",
        "default.delete": "Delete",
        "default.remove": "Remove",
        "default.save": "Save",
      }
    }
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <Component>
      <div
        className="pf-l-bullseye"
      >
        <Component>
          <div
            className="pf-c-empty-state"
          >
            <div
              className="pf-c-empty-state__content"
            >
              <Component
                icon={[Function]}
              >
                <CubesIcon
                  aria-hidden="true"
                  className="pf-c-empty-state__icon"
                  color="currentColor"
                  noVerticalAlign={false}
                  size="sm"
                >
                  <svg
                    aria-hidden="true"
                    aria-labelledby={null}
                    className="pf-c-empty-state__icon"
                    fill="currentColor"
                    height="1em"
                    role="img"
                    style={
                      Object {
                        "verticalAlign": "-0.125em",
                      }
                    }
                    viewBox="0 0 512 512"
                    width="1em"
                  >
                    <path
                      d="M488.6 250.2L392 214V105.5c0-15-9.3-28.4-23.4-33.7l-100-37.5c-8.1-3.1-17.1-3.1-25.3 0l-100 37.5c-14.1 5.3-23.4 18.7-23.4 33.7V214l-96.6 36.2C9.3 255.5 0 268.9 0 283.9V394c0 13.6 7.7 26.1 19.9 32.2l100 50c10.1 5.1 22.1 5.1 32.2 0l103.9-52 103.9 52c10.1 5.1 22.1 5.1 32.2 0l100-50c12.2-6.1 19.9-18.6 19.9-32.2V283.9c0-15-9.3-28.4-23.4-33.7zM358 214.8l-85 31.9v-68.2l85-37v73.3zM154 104.1l102-38.2 102 38.2v.6l-102 41.4-102-41.4v-.6zm84 291.1l-85 42.5v-79.1l85-38.8v75.4zm0-112l-102 41.4-102-41.4v-.6l102-38.2 102 38.2v.6zm240 112l-85 42.5v-79.1l85-38.8v75.4zm0-112l-102 41.4-102-41.4v-.6l102-38.2 102 38.2v.6z"
                      transform=""
                    />
                  </svg>
                </CubesIcon>
              </Component>
              <Component
                headingLevel="h5"
                size="lg"
              >
                <h5
                  className="pf-c-title pf-m-lg"
                >
                  <FormattedMessage
                    defaultMessage="No vulnerability data"
                    id="emptyState.noData"
                    values={Object {}}
                  >
                    No vulnerability data
                  </FormattedMessage>
                </h5>
              </Component>
              <Component>
                <div
                  className="pf-c-empty-state__body"
                >
                  <FormattedMessage
                    defaultMessage="Activate the Insights client for this system to report for vulnerabilities"
                    id="emptyState.activateInsights"
                    values={Object {}}
                  >
                    Activate the Insights client for this system to report for vulnerabilities
                  </FormattedMessage>
                </div>
              </Component>
              <Component
                component="a"
                href="http://access.redhat.com/products/cloud_management_services_for_rhel#getstarted"
                variant="primary"
              >
                <a
                  aria-disabled={false}
                  aria-label={null}
                  className="pf-c-button pf-m-primary"
                  data-ouia-component-id={null}
                  data-ouia-component-type="PF4/Button"
                  data-ouia-safe={true}
                  disabled={null}
                  href="http://access.redhat.com/products/cloud_management_services_for_rhel#getstarted"
                  tabIndex={null}
                  type={null}
                >
                  <FormattedMessage
                    defaultMessage="Learn about the Insights client"
                    id="emptyState.learnInsights"
                    values={Object {}}
                  >
                    Learn about the Insights client
                  </FormattedMessage>
                </a>
              </Component>
            </div>
          </div>
        </Component>
      </div>
    </Component>
  </IntlProvider>
</IntlProvider>
`;

exports[`Empty state should render FilterNotFoundForCVE 1`] = `
<IntlProvider
  locale="en"
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="en"
    messages={
      Object {
        "default.cancel": "Cancel",
        "default.delete": "Delete",
        "default.remove": "Remove",
        "default.save": "Save",
      }
    }
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <Component
      style={
        Object {
          "height": "auto",
        }
      }
    >
      <div
        className="pf-l-bullseye"
        style={
          Object {
            "height": "auto",
          }
        }
      >
        <Component
          variant="large"
        >
          <div
            className="pf-c-empty-state pf-m-lg"
          >
            <div
              className="pf-c-empty-state__content"
            >
              <Component
                headingLevel="h5"
                size="lg"
              >
                <h5
                  className="pf-c-title pf-m-lg"
                >
                  <FormattedMessage
                    defaultMessage="No matching CVEs found"
                    id="emptyState.noMatchingCves"
                    values={Object {}}
                  >
                    No matching CVEs found
                  </FormattedMessage>
                </h5>
              </Component>
              <Component>
                <div
                  className="pf-c-content"
                >
                  <Component
                    style={
                      Object {
                        "whiteSpace": "normal",
                      }
                    }
                  >
                    <div
                      className="pf-c-empty-state__body"
                      style={
                        Object {
                          "whiteSpace": "normal",
                        }
                      }
                    >
                      <FormattedMessage
                        defaultMessage="This may be for one of the following reasons:"
                        id="emptyState.maybeFollowing"
                        values={Object {}}
                      >
                        This may be for one of the following reasons:
                      </FormattedMessage>
                      <Component>
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="The criteria/filters you’ve specified result in no/zero CVEs being reported in your environment"
                            id="emptyState.noCveReported"
                            values={Object {}}
                          >
                            The criteria/filters you’ve specified result in no/zero CVEs being reported in your environment
                          </FormattedMessage>
                        </p>
                      </Component>
                      <Component
                        component="p"
                      >
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="If you think these systems have applicable CVEs, or would like more information, please contact    "
                            id="emptyState.forMoreInfo"
                            values={Object {}}
                          >
                            If you think these systems have applicable CVEs, or would like more information, please contact    
                          </FormattedMessage>
                          <a
                            href="https://access.redhat.com/security/team/contact/"
                            rel="noopener noreferrer"
                            target="__blank"
                          >
                            <FormattedMessage
                              defaultMessage="Red Hat Products Security"
                              id="emptyState.forMoreLink"
                              values={Object {}}
                            >
                              Red Hat Products Security
                            </FormattedMessage>
                              
                            <ExternalLinkAltIcon
                              color="currentColor"
                              noVerticalAlign={false}
                              size="sm"
                            >
                              <svg
                                aria-hidden={true}
                                aria-labelledby={null}
                                fill="currentColor"
                                height="1em"
                                role="img"
                                style={
                                  Object {
                                    "verticalAlign": "-0.125em",
                                  }
                                }
                                viewBox="0 0 512 512"
                                width="1em"
                              >
                                <path
                                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                                  transform=""
                                />
                              </svg>
                            </ExternalLinkAltIcon>
                          </a>
                        </p>
                      </Component>
                    </div>
                  </Component>
                </div>
              </Component>
            </div>
          </div>
        </Component>
      </div>
    </Component>
  </IntlProvider>
</IntlProvider>
`;

exports[`Empty state should render FilterNotFoundForSystem 1`] = `
<IntlProvider
  locale="en"
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="en"
    messages={
      Object {
        "default.cancel": "Cancel",
        "default.delete": "Delete",
        "default.remove": "Remove",
        "default.save": "Save",
      }
    }
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <Component
      style={
        Object {
          "height": "auto",
        }
      }
    >
      <div
        className="pf-l-bullseye"
        style={
          Object {
            "height": "auto",
          }
        }
      >
        <Component
          variant="large"
        >
          <div
            className="pf-c-empty-state pf-m-lg"
          >
            <div
              className="pf-c-empty-state__content"
            >
              <Component
                headingLevel="h5"
                size="lg"
              >
                <h5
                  className="pf-c-title pf-m-lg"
                >
                  <FormattedMessage
                    defaultMessage="No CVEs reported for this system"
                    id="emptyState.noCvesReported"
                    values={Object {}}
                  >
                    No CVEs reported for this system
                  </FormattedMessage>
                </h5>
              </Component>
              <Component>
                <div
                  className="pf-c-content"
                >
                  <Component
                    style={
                      Object {
                        "whiteSpace": "normal",
                      }
                    }
                  >
                    <div
                      className="pf-c-empty-state__body"
                      style={
                        Object {
                          "whiteSpace": "normal",
                        }
                      }
                    >
                      <Component>
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="The criteria/filters you’ve specified result in no/zero CVEs being reported in your environment"
                            id="emptyState.noCveReported"
                            values={Object {}}
                          >
                            The criteria/filters you’ve specified result in no/zero CVEs being reported in your environment
                          </FormattedMessage>
                        </p>
                      </Component>
                      <Component
                        component="p"
                      >
                        <p
                          className=""
                          data-pf-content={true}
                        >
                          <FormattedMessage
                            defaultMessage="If you think this system has applicable CVEs, or would like more information, please contact   "
                            id="emptyState.ifCveForSystem"
                            values={Object {}}
                          >
                            If you think this system has applicable CVEs, or would like more information, please contact   
                          </FormattedMessage>
                          <a
                            href="https://access.redhat.com/security/team/contact/"
                            rel="noopener noreferrer"
                            target="__blank"
                          >
                            <FormattedMessage
                              defaultMessage="Red Hat Products Security"
                              id="emptyState.forMoreLink"
                              values={Object {}}
                            >
                              Red Hat Products Security
                            </FormattedMessage>
                              
                            <ExternalLinkAltIcon
                              color="currentColor"
                              noVerticalAlign={false}
                              size="sm"
                            >
                              <svg
                                aria-hidden={true}
                                aria-labelledby={null}
                                fill="currentColor"
                                height="1em"
                                role="img"
                                style={
                                  Object {
                                    "verticalAlign": "-0.125em",
                                  }
                                }
                                viewBox="0 0 512 512"
                                width="1em"
                              >
                                <path
                                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                                  transform=""
                                />
                              </svg>
                            </ExternalLinkAltIcon>
                          </a>
                        </p>
                      </Component>
                    </div>
                  </Component>
                </div>
              </Component>
            </div>
          </div>
        </Component>
      </div>
    </Component>
  </IntlProvider>
</IntlProvider>
`;
